internal/pkg/deploy/deploy.go

```diff
@@ -121,25 +121,47 @@ func (s *Store) ListEnvironmentsDeployedTo(appName string, svcName string) ([]st
 	if err != nil {
 		return nil, fmt.Errorf("list environment for app %s: %w", appName, err)
 	}
-	var envsWithDeployment []string
+	type result struct {
+		name string
+		err  error
+	}
+	deployedEnv := make(chan result, len(envs))
 	for _, env := range envs {
-		err := s.newRgClientFromRole(env.ManagerRoleARN, env.Region)
-		if err != nil {
-			return nil, err
-		}
-		resources, err := s.rgClient.GetResourcesByTags(ecsServiceResourceType, map[string]string{
-			AppTagKey:     appName,
-			EnvTagKey:     env.Name,
-			ServiceTagKey: svcName,
-		})
-		if err != nil {
-			return nil, fmt.Errorf("get resources by Copilot tags: %w", err)
+		go func(env *config.Environment) {
```

Comment:
hmm dumb question do we need to pass `env` as a parameter? I think it might already be in scope so we can do `go func(){ ... }()`

Reply:
my linter prompting me to do the change...